README.md

```diff
@@ -171,6 +171,32 @@ And give that array to Grover:
 Grover.new('<some URI with cookies', cookies: header_cookies).to_pdf
 ```
 
+#### Adding style tags
+To add style tags, pass an array of style tag options as such
+See [page.addStyleTag](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageaddstyletagoptions) documentation for more details.
+
+```ruby
+style_tag_options = [
+  { url: 'http://example.com/style.css' },
+  { path: 'style.css' },
+  { content: '.body{background: red}' }
+]
+Grover.new('<html><body><h1>Heading</h1></body></html>', style_tag_options: style_tag_options).to_pdf
+```
+
+#### Adding script tags
+To add script tags, pass an array of script tag options as such
+See [page.addScriptTag](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageaddscripttagoptions) documentation for more details.
+
+```ruby
+script_tag_options = [
+  { url: 'http://example.com/script.js' },
+  { path: 'script.js' },
+  { content: 'document.querySelector('h1').style.display = "none"' }
```

Comment:
Can you fix up your example? There are unescaped single quotes around the `h1`. Suggest wrap the `h1` with double quotes instead.

Reply:
Thanks, updated.